script.js

```diff
@@ -0,0 +1,62 @@
+const PASSWORD_BOX = document.getElementById("password");
+const LENGTH = 15;
+const UPPER_CASE = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+const LOWER_CASE = "abcdefhijklmnopqrstuvwxyz";
+const NUMBER = "0123456789";
+const SYMBOL = "!@#$%^&*()_+~";
+const ALL_CHARS = UPPER_CASE + LOWER_CASE + NUMBER + SYMBOL;
+const ELEMENTS_IDS = {
+    title: "title",
+    headerPrefix: "headerPrefix",
+    headerSuffix: "headerSuffix",
+    buttonGeneratePassword: "buttonGeneratePassword"
+};
+const LABELS = {
+    [ELEMENTS_IDS.title]: "Random Password Generator",
+    [ELEMENTS_IDS.headerPrefix]: "Generate a",
+    [ELEMENTS_IDS.headerSuffix]: "RANDOM PASSWORD",
+    placeholderPassword: "password",
+    [ELEMENTS_IDS.buttonGeneratePassword]: "Generate password"
+};
+
+function createPassword(){
```

Comment:
create extra function called `handleClickButtonGeneratePassword`
and inside of the handler, call `createPassword();`

```javascript
function handleClickButtonGeneratePassword(){
  createPassword();
}
```

Reply:
doneee